usagerep/usagerep.go

```diff
@@ -89,7 +97,7 @@ func (u *UsageReporter) PeriodicallyReportUsage(ctx context.Context) {
 
 	doReport := func() {
 		alpEnabled := (config["PolicySyncPathPrefix"] != "")
-		u.reportUsage(config["ClusterGUID"], config["ClusterType"], config["CalicoVersion"], alpEnabled, stats)
+		u.reportUsage(config["ClusterGUID"], config["ClusterType"], config["CalicoVersion"], u.staticItems.KubernetesVersion, alpEnabled, stats)
```

Comment:
I would prefer leaving this call as it was, because the `reportUsage` can access `u.staticItems` itself.